### ISSUE TITLE
JS is not copied to new colony directory

### ISSUE THREAD

**Issue by kirbysayshi** [hughsk/colony#5]
If I do:

```
$ npm install -g colony
...
$ cd my-project
$ colony index.js -o colony
path.existsSync is now called `fs.existsSync`.
Done!
$ ls -la colony
total 96
drwxr-xr-x   6 drewp  staff    204 Nov  7 19:50 .
drwxr-xr-x  21 drewp  staff    714 Nov  8 13:51 ..
drwxr-xr-x   4 drewp  staff    136 Nov  7 19:50 css
drwxr-xr-x   4 drewp  staff    136 Nov  7 19:50 files
drwxr-xr-x   5 drewp  staff    170 Nov  7 19:50 img
-rw-r--r--   1 drewp  staff  48288 Nov  7 19:50 index.html
```

For some reason, there is no `js` dir, meaning no compiled JS and thus the simulation/visualization fails. 

Am I doing something wrong?

**Comment by funston** [comment]
Ever resolve this? Seeing the same thing.

**Comment by kirbysayshi** [comment]
Unfortunately I haven't had time to dig into the code, so I don't have a workaround right now.

**Comment by brianleroux** [comment]
ya just ran into this too

**Comment by hughsk** [comment]
@brianleroux thanks for the bump, had missed this one before.

Looks like a mixup while publishing 0.0.4, and fixed for me by republishing as 0.0.5. Sorry about the delay! Closing, but feel free to reopen if you guys are still having this problem :)
